src/Workspaces/VisualBasic/Portable/Simplification/Reducers/VisualBasicExtensionMethodReducer.vb

```diff
@@ -29,7 +29,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Simplification
 
             Dim rewrittenNode = invocationExpression
 
-            If invocationExpression.Expression.Kind = SyntaxKind.SimpleMemberAccessExpression Then
+            If invocationExpression.Expression?.Kind = SyntaxKind.SimpleMemberAccessExpression Then
```

Comment:
this happens in tests, which explicitly call into the simplifier with specific subnodes.  in the test we call on an invocation expression with a null expression child.